internal/utils/utils.go

```diff
@@ -19,3 +20,100 @@ func S(n int) string {
 var TimeNowFunc = func() time.Time {
 	return time.Now()
 }
+
```

Comment:
mega-`nit`:
- Consider creating a new file for your new formatter utilities - e.g., `formatters.go`
- Adding a new set of tools - especially if they focus on a single, new concern - into something as abstract a `utils.go` is a pet peeve of mine.

Reply:
will look